cupy/cuda/function.pyx

```diff
@@ -165,15 +166,21 @@ cdef class Function:
             max(1, block[0]), max(1, block[1]), max(1, block[2]),
             args, shared_mem, s)
 
+    cpdef launch(
+            self, args, size_t gridx, size_t blockx, size_t shared_mem=0,
+            stream=None):
+        _launch(
+            self.ptr, gridx, 1, 1, blockx, 1, 1,
```

Comment:
Do you think we don't have to expose y and z as well?

Reply:
We can specify them by using `Funciton.__call__`.